tensordict/prototype/tensorclass.py

```diff
@@ -198,7 +202,9 @@ def wrapper(self, *args, batch_size, device=None, **kwargs):
         self.tensordict = TensorDict(
```

Comment:
this should be private

Reply:
Currently, we try to access the tensordict in some of the test cases. @vmoens / @tcbegley Do we want to make them private ?

Reply:
I personally would be in favour of making them private.

If we do want public access to the underlying tensordict it would be better to add a property and prevent users from writing directly to it.